byudml/imputer/random_sampling_imputer.py

```diff
@@ -13,17 +14,28 @@
 Outputs = container.pandas.DataFrame
 
 
+class Hyperparams(hyperparams.Hyperparams):
+    drop_cols_all_unknown_vals = hyperparams.Enumeration[bool](
```

Comment:
I think D3M would want us to use `hyperparams.UniformBool` instead of `hyperparams.Enumeration[bool]` here; that seems to be the convention.

Reply:
I decided to just add it to the PR myself to save Erik the hassle.